numpy/f2py/tests/test_array_from_pyobj.py

```diff
@@ -3,9 +3,8 @@
 import copy
 import pytest
 
-from numpy import (
-    array, alltrue, ndarray, zeros, dtype, intp, clongdouble
-    )
+import numpy as np
+
```

Comment:
Is this change required to avoid the deprecation warning? Does that mean all user code must now change?

Reply:
No, don't remember why I did it here, but it is just maintanence, there must have been an issue with dtype being overshadowed somewhere.